core/revapi.json

```diff
@@ -700,7 +700,49 @@
         "newArchive": "com.datastax.oss:java-driver-core:jar:4.0.0-rc1-SNAPSHOT",
         "elementKind": "method",
         "justification": "JAVA-2077: Allow reconnection policy to detect first connection attempt"
-      }
+      },
+      {
+        "code": "java.method.numberOfParametersChanged",
```

Comment:
do we need to make it not backward compatible change? We may introduce `newDelay()` method with parameter while leaving old parameterless version. I am curious what are our guidelines - when we can introduce non-compatible change?